src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--fri calendar--month-length--29">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--fri calendar--month-length--29">
```
It should work only for 28-31 modifiers
Add a modifier month-length for the calendar with values 28, 29, 30 and 31 (use @for) 
But it should show all the days when someone put some number less than 28 or greater than 31
So you need to change logic of this modifier(it should set display: none) and by default, calendar__day should be display: block.